memgpt/server/rest_api/interface.py

```diff
@@ -500,7 +500,7 @@ def function_message(self, msg: str, msg_obj: Optional[Message] = None):
 
                     processed_chunk = {
                         "function_call": {
-                            # "id": function_call.id,
+                            "id": function_call.id,
```

Comment:
@goetzrobin this was required to get some of the zod/object logic to work - basically there was a mistake and this `id` is nested for when `stream_tokens` is true, but unnested when it's false.

The `refactor-api` branch updates the messages API to use pydantic models so this should get cleaned up.